services/121-service/package.json

```diff
@@ -13,8 +13,8 @@
     "lint": "eslint \"**/*.{ts,js}\"",
     "fix": "npm run lint -- --fix",
     "build": "nest build -p tsconfig.json",
-    "start": "GLOBAL_121_VERSION=$(test -f \"VERSION.txt\" && cat VERSION.txt || echo $GLOBAL_121_VERSION)  node dist/main.js",
-    "start:dev": "nodemon --watch src/**/*.json --exec npm run start:dev:nest",
+    "start": "GLOBAL_121_VERSION=$(test -f \"VERSION.txt\" && cat VERSION.txt || echo $GLOBAL_121_VERSION) node --max-old-space-size=4096 dist/main.js",
+    "start:dev": "NODE_OPTIONS='--max-old-space-size=4096' nodemon --watch src/**/*.json --exec npm run start:dev:nest",
```

Comment:
Following up from my question from standup: how did you come up with `4096` specifically? Trial and error? Or...?

Reply:
Actually I got that we have 16GB of shared memory on azure server, So honestly speaking I tried to assign 1/4 of it and then do some tests to adjust by using trial and error. During tests, got that, for 1.5 millions 4096 is enough. I also tried with 3072 but was not enough. 

And I got in our NLRC has ~ 700,000 rows that failed during export, so wanted to make it work for double number of rows ~1.5 millions.

Reply:
I imagine we can assign more than 4GB if the total is 16GB - but I'm not the expert here.

I'm also wondering, do you know what the default is without setting this flag? The docs doesn't say anything about this.

Reply:
@aberonni I could only find documentation about the Windows-version of Azure App Services:

https://learn.microsoft.com/en-us/azure/app-service/configure-custom-container?pivots=container-windows&tabs=debian#customize-container-memory

There the defaults are mentioned as `1536` Mb for our `P2v3`-tier.

Underneath that table is:

> For example, in an App Service plan with 8 GB RAM, the cumulative total of `WEBSITE_MEMORY_LIMIT_MB` for all the apps must not exceed 8 GB.

So that would limit us to "only make an exception to this limit for a few clients, at the expense of the others". Could still be worth it, for now. But not a good, solid path going forward, I'd think.

But we could try this out "in production" on the Staging-environment...
To see what actually happens...

Reply:
Another idea:

We could also set these ENV-options _only_ in the NLRC-AppService...
Or divide the 16Gbs up into "unequal"-chunks and set the ENV-options on all AppServices (but with different numbers for each)...

That way, we don't have to hard-code anything for production...